cluster.go

```diff
@@ -0,0 +1,95 @@
+package allocator
+
+// Resource is 32-bit identifier for resources -> {disk, QPS}
+type Resource int
```

Comment:
`Resource` is currently only used be `cluster.go` and `node.go`, but is exposed globally, would be good if we can use it only for these types.

Reply:
Also by allocator.go, forgot about that one.